synapse/handlers/sliding_sync.py

```diff
@@ -839,9 +842,13 @@ async def handle_room(room_id: str) -> None:
             with start_active_span("sliding_sync.generate_room_entries"):
                 await concurrently_execute(handle_room, relevant_rooms_to_send_map, 10)
 
+        new_connection_state = previous_connection_state.get_mutable()
+
         extensions = await self.get_extensions_response(
             sync_config=sync_config,
             actual_lists=lists,
+            previous_connection_state=previous_connection_state,
+            new_per_connection_state=new_connection_state,
```

Comment:
To match `previous_connection_state`:

```suggestion
            new_connection_state=new_connection_state,
```

(throughout)